src/components/SegmentedRadioGroup/SegmentedRadioGroupContext.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+import type {UseRadioGroupContextProps} from '../../hooks/private/useRadioGroup/types';
+
+export const SegmentedRadioGroupContext = React.createContext<UseRadioGroupContextProps>({
+    name: '',
+    currentValue: '',
+    disabled: false,
+    ref: () => {},
+    onChange: () => {},
+});
```

Comment:
Let's make the default value of the context to be null. And throw on incorrect usage: https://github.com/gravity-ui/uikit/blob/main/src/components/tabs/hooks/useTab.ts#L14